vlingo-reactive-messaging-patterns/src/main/java/io/vlingo/reactive/messaging/patterns/recipientlist/PriceQuoteInterest.java

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+package io.vlingo.reactive.messaging.patterns.recipientlist;
+
+/**
+ * PriceQuoteInterest used to register interest in providing {@link PriceQuote} for each retail item where appropriate.
+ *
+ * @author brsg.io
+ * @since Nov 20, 2018
+ */
+public class PriceQuoteInterest
+{
+    public final String path;
```

Comment:
Seems like `path` is implementation specific and maybe not a useful name in this case. From above observations I suggest `type` rather than `path`.